### PR TITLE
Swift 2

### PR DESCRIPTION
**Pull request by madhikarma**
### Description
- Conversion to Swift 2 using Xcode 7.0
- Updated project settings using Xcode 7.0
### Testing notes
- Tested using iOS 9 and iOS 8 and no signs of regression found

### CODE REVIEW

File: AutoLayoutHelper.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -346,6 +347,7 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				INFOPLIST_FILE = AutoLayoutHelper/Info.plist;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
+				PRODUCT_BUNDLE_IDENTIFIER = "com.madhikarma.$(PRODUCT_NAME:rfc1034identifier)";
```
Comment (danieladias) [inline AutoLayoutHelper.xcodeproj/project.pbxproj]: Is this a good bundle identifier?

File: AutoLayoutHelper.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -356,6 +358,7 @@
...
```
Comment (danieladias) [inline AutoLayoutHelper.xcodeproj/project.pbxproj]: Same as above

File: AutoLayoutHelper.xcodeproj/project.pbxproj (hunk 1 above)
Comment (madhikarma) [inline AutoLayoutHelper.xcodeproj/project.pbxproj]: Nope :) I ran a project settings update using Xcode 7.0 and it inserted this for me so i'm guessing its either  
- using a default organisation / company setting 
- this was something originally set in the project i.e. before it was forked to ustwo from my personal account 

screen shot 2015-10-14 at 10 18 23

Comment (madhikarma) [inline AutoLayoutHelper.xcodeproj/project.pbxproj]: Fixed

screen shot 2015-10-14 at 10 26 45

File: AutoLayoutHelper.xcodeproj/project.pbxproj (hunk 2 above)
Comment (madhikarma) [inline AutoLayoutHelper.xcodeproj/project.pbxproj]: Fixed

File: AutoLayoutHelper.xcodeproj/project.pbxproj (hunk 1 above)
Comment (madhikarma) [inline AutoLayoutHelper.xcodeproj/project.pbxproj]: I looked through the original diff and it was previously set as com.madhikarma so nothing to do with Xcode 7 just something I didn't realise when we brought it back to ustwo.

fyi: nNext Pull Request will add a target to create a dynamic framework so its easier to integrate

**Pull request merged**